### DISCUSSION TITLE
Using Event Transform to customize calendar symbols

### DISCUSSION THREAD

**Discussion by jonwinny** [MMRIZE/MMM-CalendarExt3Journal#3] [General]
Hi!

I am trying to use the event transform function to customize the symbols my calendar displays based on words in the event title. I have followed the documentation format but all I am getting is the symbol set in the default calendar function. I have been able to edit the color of events with the same structure, but symbol is not working. Code snippet follows:

```
eventTransformer: (ev) => {
						if(ev.title.search('foo') > -1) ev.color = "#002F6C"
						if(ev.title.search('foobar') > -1) ev.color = "#872736"
						if(ev.title.search('poobar') > -1) ev.color = "#c1c2ff"
						if(ev.title.search('poorbar') > -1) ev.symbol = ["baseball"]
						return ev
					},
```
Any help appreciated!

**Comment by eouia** [comment]
<img width="866" alt="image" src="https://github.com/MMRIZE/MMM-CalendarExt3Journal/assets/1720610/5cb8a32e-c54d-4681-b16f-2985657aff21">

Due to the weird broadcasting format of the default calendar module, you should do a favour a little more.
```js
eventTransformer: (ev) => {
	if (ev.title.search('Repeated') !== -1) ev.symbol = ['fas fa-fw fa-baseball']
	return ev
}
```
